src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -129,22 +133,58 @@ public static void logOnLowLevel(String command, String description, boolean suc
     log(command, description, success, true);
   }
 
+  //TODO: Check error class here
   private static void log(String command, String description, boolean isSuccess,
-      boolean ifLowLevel) {
+                   boolean ifLowLevel) {
     logsResults.add(isSuccess);
     String escapedDescription = escapeHtml(description);
 
-    String className = isSuccess ? "success" : "error";
-    StringBuilder builder = new StringBuilder();
-    if (ifLowLevel) {
-      builder.append("<tr class=\"" + className + " lowLevelAction" + "\"><td>" + command
-          + "</td><td>" + escapedDescription + "</td><td> <br/> &nbsp;</td></tr>");
-    } else {
-      builder.append("<tr class=\"" + className + "\"><td>" + command + "</td><td>"
-          + escapedDescription + "</td><td> <br/> &nbsp;</td></tr>");
+    String className = isSuccess ? SUCCESS_CLASS : ERROR_CLASS;
+//    if(!isSuccess){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)